MonolithDev/gettingSongs/dj_mix_finder.py

```
@@ -0,0 +1,245 @@
+        """
+        track_info = result.track_info
+        queries = result.generated_queries
+        
+        print(f"\n{'='*60}")
+        print(f"TRACK: {track_info['title']}")
+        print(f"ARTIST(S): {', '.join(track_info['artists'])}")
+        print(f"ALBUM: {track_info['album']}")
+        print(f"YEAR: {track_info['release_year']}")
+        print(f"{'='*60}")
+        
+        print(f"\nREASONING:")
```

Ruff (0.13.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)